packages/web/src/components/notification/Notifications/ProfilePicture.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useCallback, useEffect, useState } from 'react'
+
+import { push } from 'connected-react-router'
+import { useDispatch } from 'react-redux'
+
+import { SquareSizes } from 'common/models/ImageSizes'
+import { User } from 'common/models/User'
+import ArtistPopover from 'components/artist/ArtistPopover'
+import DynamicImage from 'components/dynamic-image/DynamicImage'
+import { useUserProfilePicture } from 'hooks/useUserProfilePicture'
+
+import styles from './ProfilePicture.module.css'
+
+const imageLoadDelay = 250
+
+type ProfilePictureProps = {
+  user: User
+  className?: string
+  disablePopover?: boolean
+}
+
+export const ProfilePicture = (props: ProfilePictureProps) => {
```

Comment:
hmm looks like i should be able to use this component in many of the tipping components im making. nice

Reply:
Sweet, I'll move both of these to components directory in the next few prs